### ISSUE TITLE
cffi build is failing on github actions/build_wasm

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#2544]
```
Collecting cffi
  Downloading cffi-1.15.1.tar.gz (508 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 508.5/508.5 kB 16.0 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'

--- Stderr:
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      Traceback (most recent call last):
        File "/tmp/build-env-4nmm04r3/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/tmp/build-env-4nmm04r3/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/tmp/build-env-4nmm04r3/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-hbzzts93/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 338, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
        File "/tmp/pip-build-env-hbzzts93/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 320, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-hbzzts93/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 484, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-hbzzts93/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 148, in <module>
        File "<string>", line 75, in ask_supports_thread
        File "<string>", line 70, in get_config
        File "/tmp/pip-build-env-hbzzts93/overlay/lib/python3.10/site-packages/setuptools/_distutils/sysconfig.py", line 544, in get_config_vars
          _config_vars = sysconfig.get_config_vars().copy()
        File "/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10/sysconfig.py", line 615, in get_config_vars
          _init_posix(_CONFIG_VARS)
        File "/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10/sysconfig.py", line 477, in _init_posix
          _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
      ModuleNotFoundError: No module named '_sysconfigdata__emscripten_wasm32-emscripten'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by ctb** [comment]
similar error mentioned here: https://github.com/pyodide/pyodide/discussions/2729

**Comment by luizirber** [comment]
Fixed by updating to pyodide 0.23.0 + Python 3.11: https://github.com/sourmash-bio/sourmash/pull/2545
